{"uuid":"c37f8b20-4496-46a6-9496-7ce2f6670006","historyId":"f88b279a73b66872ca22cbf9256fc169","fullName":"com.testingacademy.tests.VWOLogin.testVwoLoginNegative","labels":[{"name":"package","value":"com.testingacademy.tests.VWOLogin"},{"name":"testClass","value":"com.testingacademy.tests.VWOLogin"},{"name":"testMethod","value":"testVwoLoginNegative"},{"name":"parentSuite","value":"All Test Suite"},{"name":"suite","value":"/FirstAutomationFrameWork"},{"name":"subSuite","value":"com.testingacademy.tests.VWOLogin"},{"name":"host","value":"DESKTOP-T2H52KL"},{"name":"thread","value":"25924@DESKTOP-T2H52KL.main(1)"},{"name":"framework","value":"testng"},{"name":"language","value":"java"}],"links":[],"name":"testVwoLoginNegative","status":"skipped","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"java.io.FileNotFoundException: C:\\Users\\Naman\\IdeaProjects\\FirstAutomationFrameWorksrc\\main\\resources\\data.properties (The system cannot find the path specified)","trace":"java.lang.RuntimeException: java.io.FileNotFoundException: C:\\Users\\Naman\\IdeaProjects\\FirstAutomationFrameWorksrc\\main\\resources\\data.properties (The system cannot find the path specified)\r\n\tat com.testingacademy.utils.PropertyReader.readKey(PropertyReader.java:22)\r\n\tat com.testingacademy.driver.DriverManager.init(DriverManager.java:24)\r\n\tat com.testingacademy.tests.VWOLogin.setUP(VWOLogin.java:23)\r\n\tat java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)\r\n\tat java.base/java.lang.reflect.Method.invoke(Method.java:580)\r\n\tat org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)\r\n\tat org.testng.internal.invokers.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:69)\r\n\tat org.testng.internal.invokers.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:393)\r\n\tat org.testng.internal.invokers.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:326)\r\n\tat org.testng.TestRunner.invokeTestConfigurations(TestRunner.java:661)\r\n\tat org.testng.TestRunner.beforeRun(TestRunner.java:650)\r\n\tat org.testng.TestRunner.run(TestRunner.java:612)\r\n\tat org.testng.SuiteRunner.runTest(SuiteRunner.java:443)\r\n\tat org.testng.SuiteRunner.runSequentially(SuiteRunner.java:437)\r\n\tat org.testng.SuiteRunner.privateRun(SuiteRunner.java:397)\r\n\tat org.testng.SuiteRunner.run(SuiteRunner.java:336)\r\n\tat org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)\r\n\tat org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)\r\n\tat org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)\r\n\tat org.testng.TestNG.runSuitesLocally(TestNG.java:1228)\r\n\tat org.testng.TestNG.runSuites(TestNG.java:1134)\r\n\tat org.testng.TestNG.run(TestNG.java:1101)\r\n\tat com.intellij.rt.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:65)\r\n\tat com.intellij.rt.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:105)\r\nCaused by: java.io.FileNotFoundException: C:\\Users\\Naman\\IdeaProjects\\FirstAutomationFrameWorksrc\\main\\resources\\data.properties (The system cannot find the path specified)\r\n\tat java.base/java.io.FileInputStream.open0(Native Method)\r\n\tat java.base/java.io.FileInputStream.open(FileInputStream.java:213)\r\n\tat java.base/java.io.FileInputStream.<init>(FileInputStream.java:152)\r\n\tat java.base/java.io.FileInputStream.<init>(FileInputStream.java:106)\r\n\tat com.testingacademy.utils.PropertyReader.readKey(PropertyReader.java:18)\r\n\t... 23 more\r\n"},"stage":"finished","description":"Verify that with invalid email, pass, error message is shown on the app.vwo.com","steps":[],"attachments":[],"parameters":[],"start":1736872826214,"stop":1736872826218}